src/sentry/api/event_search.py

```diff
@@ -684,6 +684,16 @@ def to_list(value):
     # Long term these will become more complex functions but these are
     # field aliases.
     "apdex": {"result_type": "number", "aggregations": [["apdex(duration, 300)", "", "apdex"]]},
+    "impact": {
+        "result_type": "number",
+        "aggregations": [
+            [
+                "(1 - ((countIf(duration < 300) + (countIf((duration > 300) AND (duration < 1200)) / 2)) / count())) + (1 - 1 / sqrt(uniq(user))) * 3",
```

Comment:
```suggestion
                "(1 - ((countIf(duration < 300) + (countIf((duration > 300) AND (duration < 1200)) / 2)) / count())) + ((1 - 1 / sqrt(uniq(user))) * 3)",
```

@dcramer just need some more parenthesis